release_notes.md

```diff
@@ -1,7 +1,7 @@
 ## next (unreleased)
 
 - [#779](https://github.com/Flank/flank/pull/779) Print retries & display additional info. ([jan-gogo](https://github.com/jan-gogo))
-- 
+- [#796](https://github.com/Flank/flank/issues/796) Ability to generate XML results with passing re-runs removed. ([adamfilipow92](https://github.com/adamfilipow92))
```

Comment:
> Ability to generate XML results with passing re-runs removed.

[In the issue](https://github.com/Flank/flank/issues/796), the problem described is a passing flaky test will have failure nodes. What we want to do is remove failure nodes from flaky tests so that the JUnit CI plugin passes.

I'd reword this to be something like: Remove failure nodes from tests that passed on retry so that Jenkins JUnit plugin marks them as successful.

Reply:
Thanks I updated description.